README.rdoc

```diff
@@ -68,6 +68,20 @@ Running the Rake task will print something like this for you:
 
 OMG super helpful, isn't it?
 
+== How do I tell it to ignore routes?
+
+Some gems out there that inject routes or actions into our app for testing. Jasmine-Rails does this. They can give you false negatives when running this in development mode. It can be useful to ignore said routes or actions.
+
+Create a .traceroute.yaml (or .traceroute.yml or .traceroute) file in your root directory.
+
+  # .traceroute.yaml
+  ignore_unreachable_actions:
```

Comment:
Inconsistency. Here is `ignore_unreachable_actions` and in `lib/traceroute.rb` is `@ingored_unreachable_actions`